modules/stochastic_tools/examples/surrogates/poly_chaos_normal_mc.i

```diff
@@ -103,17 +73,17 @@
     order = 10
     distributions = 'k_dist q_dist L_dist Tinf_dist'
     sampler = sample
-    results_vpp = samp_avg
-    results_vector = sample
+    results_vpp = results
+    results_vector = data:avg
```

Comment:
what is this syntax?

Reply:
This is the name of the vector, `declareVector("data:avg")`

Reply:
I was hoping it was some exotic parser syntax that I didn’t know yet